PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/properties/AzResourcePropertiesEditor.java

```diff
@@ -23,11 +23,13 @@
     private final T resource;
     private final TailingDebouncer debouncer;
     protected final Project project;
+    protected final AzureResourceEditorViewManager manager;
 
     public AzResourcePropertiesEditor(@Nonnull final VirtualFile virtualFile, @Nonnull T resource, @Nonnull final Project project) {
         super(virtualFile);
         this.resource = resource;
         this.project = project;
+        this.manager = virtualFile.getUserData(AzureResourceEditorViewManager.AZURE_RESOURCE_EDITOR_MANAGER_KEY);
```

Comment:
make `AzureResourceEditorViewManager.closeEditor` a static method? It seems that no need to pass it to child editors